cmd/kubectl-who-can.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/aquasecurity/kubectl-who-can/pkg/cmd"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"os"
+)
+
+func main() {
+	root := cmd.NewCmdWhoCan()
+	if err := root.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
Should we be printing out the error?

Reply:
As far as I see it's already printed out as the first line in the output by cobra:

```
./kubectl-who-can get pod --xxx
Error: unknown flag: --xxx
Usage:
  ...
Examples:
  ...
Flags:
  ...
```

Printing it in the main function will output the error message twice:

```
./kubectl-who-can get pod --xxx
Error: unknown flag: --xxx
Usage:
  ...
Examples:
  ...
Flags:
  ...
Error: unknown flag: --xxx
```